docs/source/tutorial/dynamodb.rst

```diff
@@ -0,0 +1,293 @@
+.. _dynamodb_tutorial
+
+DynamoDB Tutorial
+=================
+This tutorial shows how to use
+`Amazon DynamoDB <http://aws.amazon.com/documentation/dynamodb/>`_ with Boto3.
+It will focus on how to use the :py:class:`DynamoDB.ServiceResource` and
```

Comment:
Instead of focusing on the specific classes, I'd prefer if said the the user is going to accomplish by following this tutorial.  "You'll create a dynamodb table and learn ..."